### ISSUE TITLE
Prices on XRP

### ISSUE THREAD

**Issue by abn2609** [Tucsky/SignificantTrades#104]
Hey Tucsky,

Can the prices be fine-tuned to utilise the full 4 decimal places, instead of showing just a flat 0.1900 until 0.2000? 5%-10% price changes are quite big to be ticks

**Comment by Tucsky** [comment]
You are absolutely right, I wasn't setting the minimum move for pricescale to be updated along with the precision (number of decimal to show). Thanks for reporting that!

So the app first check for trades on startup and see how exchanges are showing the price, then do average of the reported precisions. For xrp/btc it should show 7 or 8 decimals...
You can set the precision you want in the settings as well :)
![precision](https://user-images.githubusercontent.com/5738336/85235180-5fa13480-b413-11ea-8d3e-77c5e77f4a87.gif)

**Comment by abn2609** [comment]
No worries, thanks Kevin!

**Comment by traviskk** [comment]
😁
